app/src/main/java/org/simple/clinic/scanid/ScanSimpleIdUpdate.kt

```diff
@@ -26,6 +27,16 @@ class ScanSimpleIdUpdate @Inject constructor(
       is ShortCodeSearched -> next(model.shortCodeChanged(event.shortCode), ValidateShortCode(event.shortCode))
       is ScanSimpleIdScreenQrCodeScanned -> simpleIdQrScanned(model, event)
       is PatientSearchByIdentifierCompleted -> patientSearchByIdentifierCompleted(model, event)
+      is EnteredCodeJsonParsed -> enteredCodeParsed(model, event)
+    }
+  }
+
+  private fun enteredCodeParsed(model: ScanSimpleIdModel, event: EnteredCodeJsonParsed): Next<ScanSimpleIdModel, ScanSimpleIdEffect> {
```

Comment:
The same comment as renaming `EnteredCodeJsonParsed`, let's rename this method as well.